croud/consumers/commands.py

```diff
@@ -24,11 +24,11 @@
 from croud.util import clean_dict
 
 
-def consumer_sets_list(args: Namespace) -> None:
+def consumers_list(args: Namespace) -> None:
     body = dedent(
         """
-    query allConsumerSets($clusterId: String, $productId: String, $projectId: String) {
-        allConsumerSets(clusterId: $clusterId, productId: $productId, projectId: $projectId) {
+    query allConsumers($clusterId: String, $productId: String, $projectId: String) {
```

Comment:
Please use ID type and productId -> productName

```suggestion
    query allConsumers($clusterId: ID, $productName: String, $projectId: ID) {
```